### PR TITLE
style: redesigned hero section

### PR DESCRIPTION
**Pull request by franky47**
@peterpeterparker Let me know about the responsive size of the tagline, I tried making it always fit on two lines, but there's a range of widths (768 to 960px) where the animation on the side doesn't leave much space, should we hide it in those cases?

### CODE REVIEW

File: src/components/Hero/index.tsx [hunk 1]
```diff
@@ -11,7 +11,9 @@ export default function Hero(): JSX.Element {
     <article className={clsx("hero", styles.heroBanner)}>
       <div className={`${styles.container}`}>
         <h1 className={`hero__title ${styles.title} ${styles.item}`}>
-          {siteConfig.tagline}
+          Build Web3
+          <br />
```
Comment (peterpeterparker) [inline src/components/Hero/index.tsx:15]: It's SEO friendly to have a `br` within a title?

File: src/components/Illustration/index.tsx [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
     </div>
   );
 };
 
 export default Illustration;
+
+// --
+
+function LottieAnimation() {
+  const container = React.useRef(null);
```
Comment (peterpeterparker) [inline src/components/Illustration/index.tsx:20]: If you don't mind can you be explicit on the imports? I like to be explicit to gives hint on the bundler that we do not require an entire lib.

```
import {loadAnimation} from "lottie-web";
import {useEffect, useRef} from "react";
```

File: src/components/Illustration/index.tsx [hunk 3]
```diff
@@ -1,13 +1,38 @@
-import IllustrationSvg from "@site/static/img/illustration.svg";
+import lottie from "lottie-web";
+import React from "react";
+import animationData from "./lottie-animation.json";
```
Comment (peterpeterparker) [inline src/components/Illustration/index.tsx:3]: The JSON file gets chunked by the build right?

**Review by peterpeterparker — COMMENTED** [review]
Thanks for the PR!

> ...a range of widths (768 to 960px) ... should we hide it in those cases?

It's probably the easiest quick solution. Let's keep it hidden till it "looks good". Agree?

File: src/components/Hero/index.tsx (hunk 1 above)
Comment (franky47) [reply src/components/Hero/index.tsx:15]: They don't care much about whitespace.

File: src/components/Illustration/index.tsx (hunk 3 above)
Comment (franky47) [reply src/components/Illustration/index.tsx:3]: It gets bundled within the JS bundle, if that's your question?

Ideally, a progressive enhancement of showing the static SVG first, then lazy-loading the whole animation (lottie-web dep + JSON file) and replacing the contents once downloaded would give a better UX on initial load. I'll see what I can do once the landing page is complete.

Comment (peterpeterparker) [reply src/components/Illustration/index.tsx:3]: That would be nice!

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**